<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCameraViewfinder" name="cameraFrame" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>341</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="titlieLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>128</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>智能闸道</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="picturLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>300</width>
     <height>350</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="retultLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(186, 189, 182);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="identifyButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>200</y>
     <width>89</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(186, 189, 182);
border: 3px solid #8f8f91;</string>
   </property>
   <property name="text">
    <string>拍照识别</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header location="global">qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
